wall_e/src/WalleModels/models.py

```diff
@@ -1,11 +1,74 @@
+# for type references to own class
+# https://stackoverflow.com/a/33533514
+from __future__ import annotations
+from typing import List
 import datetime
 import time
-
 import pytz
 from asgiref.sync import sync_to_async
 from django.conf import settings
 from django.db import models
 from django.forms import model_to_dict
+from .customFields import GeneratedIdentityField
+
+
+import logging
+logger = logging.getLogger('wall_e')
+
+
+class BanRecords(models.Model):
+    ban_id = GeneratedIdentityField(primary_key=True, always=True)
+    username = models.CharField(max_length=32, null=False)
+    user_id = models.BigIntegerField(null=False)
+    mod = models.CharField(max_length=32, null=True)
+    mod_id = models.BigIntegerField(null=True)
+    ban_date = models.BigIntegerField(null=True)
+    reason = models.TextField(null=False)
+    unban_date = models.BigIntegerField(null=True, default=None)
+
+    class Meta:
+        db_table = 'WalleModels_ban_records'
+
+    @classmethod
+    @sync_to_async
+    def insert_records(cls, records: List[BanRecords]) -> None:
+        """Adds entry to BanRecords table"""
+        for record in records:
```

Comment:
look into `bulk_create` instead

Reply:
I am pretty sure that `bulk_create` makes this for loop unnecessary, no?

Reply:
lmao, i forgot to remove it. fixed it now.